SS2-Project/Assets/Starstorm2/Modules/EntityStates/LampBoss/LampBossDeath.cs

```diff
@@ -31,10 +31,6 @@ public override void OnEnter()
             hasPlayedEffect = false;
             if (characterMotor)
                 characterMotor.enabled = false;
-            //if (modelLocator && initialEffect)
-            //EffectManager.
-
-            //FindModelChild("GlowParticles").gameObject.SetActive(true);
 
             isBlue = GetModelTransform().GetComponentInChildren<ModelSkinController>().skins[characterBody.skinIndex].nameToken == "SS2_SKIN_LAMP_BLUE";
```

Comment:
Can we compare the SkinCatalog.SkinIndex of the skin instead of doing a string comparison?